test/test_helper.rb

```diff
@@ -55,9 +56,11 @@ class FakeNamespace
   def initialize(family, name, major, minor, release = nil)
     @mediapath = 'url --url http://localhost/repo/xyz'
     @root_pass = '$1$redhat$9yxjZID8FYVlQzHGhasqW/'
-    @grub_pass = 'blah'
+    @grub_pass = '--password=blah'
```

Comment:
Can you actually make this a valid password so it works even with `--crypted` option? Here is "redhat" one (surprisingly long).

'--password grub.pbkdf2.sha512.10000.A3602AF184F11072D8BB3801806D9B4F3B564F91478CB77D6F7E84E4D323CAE1DE11C8361311B890E4448892C4C64393DC65BFF5F26606A5D173B549931F2F20.A52D4290C63B6DD3EC330617AD71AFFB03C126EC075A6F5FED562898C64B7B4BD70485CD9109A1CD731D8F9F26FD601CA906BC7E4B806FCC2FF2A19DAE2E4123'

Reply:
I did notice that foreman never actually passes in just `--password`, it's always `--iscrypted --password` but EL5 doesn't like that. I guess there we pass in `--md5pass`. This was a bit of a shortcut.

Reply:
In my other branch I had to actually do this, I guess you made some change that made it working. I am fine with blah then.